app/routes.py

```diff
@@ -1 +1,208 @@
-from flask import Blueprint
\ No newline at end of file
+import requests
+from flask import Blueprint, jsonify, request, abort, make_response 
+from app import db 
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+import json
+
+
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+@tasks_bp.route("", methods=["POST"])
+def add_task():
+    request_body = request.get_json()
+
+    if "description" not in request_body\
+        or "title" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
+    
+    new_task = Task(title=request_body["title"], description=request_body["description"])
```

Comment:
Removing `completed_at` from your `Task.from_dict()` method would allow you to use it here :)